olive/model/handler/pytorch.py

```diff
@@ -315,21 +315,22 @@ def __init__(
         adapter_path: OLIVE_RESOURCE_ANNOTATIONS = None,
         model_attributes: Optional[Dict[str, Any]] = None,
     ):
+        validate_io_config = (
+            validate_config(io_config, IoConfig).dict() if isinstance(io_config, (IoConfig, dict)) else io_config
+        )
```

Comment:
it seems we can move this call into base class?

Reply:
that would work too. but we would need to update some other snpe related code to get a dict version of `io_config` since all snpe related code expect dicts.

Reply:
the `self.io_config` is always a dict in previous logics no matter the input is dict or `IoConfig`. Maybe no need to override the get_io_config I think.
`validate_config(io_config, IoConfig).dict()`

Reply:
oh yeah, I missed that part. makes it simple then.